### DISCUSSION TITLE
你好，可否增加隐藏指定路径下某一类文件，如扩展名为.jpg

### DISCUSSION THREAD

**Discussion by longli928** [AlistGo/alist#464] [Ideas]
问题背景：刮削器本地挂削后会在目录里创建.nfo，.jpg的文件，这些文件在kodi中不会显示，但在网页上.nfo，.jpg文件和视频混在一起，不容易分辨出视频；
元信息里一个个隐藏，工作量实在太大；
可否增加如隐藏.Redme文件这样的，针对指定扩展名隐藏

**Comment by xhofe** [comment]
是需要后端实际上返回，仅在web页面上隐藏的意思吗

**Reply by longli928** [reply]
感谢回复，是的，web页不显示即可
